config/middleware/receive-message/map-request-params.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const configVars = {};
+
+configVars.client = 'gambit-conversations';
+configVars.containerProperty = 'body';
+configVars.emojiStripParam = 'text';
+configVars.lowercaseParam = 'text';
+configVars.paramsMap = {
+  keyword: 'keyword',
+  broadcastId: 'broadcast_id',
+  text: 'incoming_message',
+  mediaUrl: 'incoming_image_url',
```

Comment:
I tried adding `campaignId: 'campaignId',` here to avoid creating the [`receive-message` middleware](https://github.com/DoSomething/gambit/blob/703fa6d59bde8db4f8ab47247c0e4e7e64ef4804/config/middleware/receive-message/map-request-params.js) -- but the `req.campaignId` wouldn't be defined in the [User Get middleware.](https://github.com/DoSomething/gambit/blob/703fa6d59bde8db4f8ab47247c0e4e7e64ef4804/lib/middleware/user-get.js#L13)